### DISCUSSION TITLE
404 on images in /_next/images

### DISCUSSION THREAD

**Discussion by adrienharnay** [vercel/next.js#29709] [Help]
### What version of Next.js are you using?

11.1.3-canary.48

### What version of Node.js are you using?

16.6.1

### What browser are you using?

Chrome

### What operating system are you using?

macOS

### How are you deploying your application?

next start

### Describe the Bug

Hi,

I have lots of `.svg` files imported in my project, and they are moved to the `.next/images` folder. When running my app, I have a 404 on `http://localhost:3000/_next/images/my-icon.svg`, but `open .next/images/my-icon.svg` correctly opens the icon in Chrome.

I don't understand why the 404 and believe there is a bug somewhere.

### Expected Behavior

Images in `.next/images` don't result in 404.

### To Reproduce

Handle SVG imports
```
    webpack: (config) => {
      config.module.rules.push({
        test: /\.svg$/i,
        issuer: { and: [/\.(ts|md)x?$/] },
        use: [
          {
            loader: 'file-loader',
          },
        ],
      });

      return config;
    },
```

Import SVG
```
import MyIcon from './my-icon.svg';
```

Use SVG
```
<img src={MyIcon} />
```

**Comment by ijjk** [comment]
Hi, you don't need to use `file-loader` when importing images in Next.js now, please remove it in favor of the built-in import handling, related documentation [here](https://nextjs.org/docs/basic-features/image-optimization#local-images)
